### ISSUE TITLE
CookieJar cannot be passed at axios@0.19.0

### ISSUE THREAD

**Issue by bluelovers** [3846masa/axios-cookiejar-support#183]
```ts
import _axios from 'axios';
import axiosCookieJarSupport from 'axios-cookiejar-support';
import { LazyCookieJar } from 'lazy-cookies';
import toughCookie from 'tough-cookie';

export const axios = axiosCookieJarSupport(_axios) as typeof _axios;

let a = axios.create({
	//jar: new LazyCookieJar(),
	jar: new toughCookie.CookieJar(),
});

console.dir(a.defaults.jar);
// => undefined
```

**Comment by 3846masa** [comment]
Thanks for reporting.
It seems to be caused because of https://github.com/axios/axios/commit/72c66dfdecbcc165c52fcd19d1f8444ad8c011c6.

It is difficult to fix because `mergeConfig` disallow non-original parameter.
I will fix this issue asap, however it takes few days.

If you want to use now, please use axios@0.18.2 until solving this issue.

**Comment by bluelovers** [comment]
i use stupid way, set it again after create

**Comment by 3846masa** [comment]
Fixed as axios-cookiejar-support@0.5.0
